docs/source/user_guide/advanced.rst

```diff
@@ -9,7 +9,9 @@ creating a new instance of ``configsuite.BasicType``. The constructor takes a
 type name, as well as a validator of the type. For instance, the ``Date``
 type in Config Suite could be implemented as follows:
 
-.. code-block:: python
+.. testcode:: [user_types]
+
+    import configsuite
```

Comment:
```suggestion
    import configsuite
    import datetime
```

Such that `_is_date` actually can be tested.

Reply:
Thanks for the thorough review 👍